client/scss/base/_layout.scss

```diff
@@ -6,6 +6,22 @@
   }
 }
 
+// sticky footer
+html {
+  height: 100%;
+}
+
+body {
+  min-height: 100%;
+  display: flex;
```

Comment:
would `min-height: 100vh` be better? I think I've seen that recommended as a 'sticky footer solved by flexbox' solution.